src/emulator/gui/src/gui.cpp

```diff
@@ -142,9 +142,9 @@ static void init_font(State &gui) {
     gui.normal_font = io.Fonts->AddFontFromMemoryTTF(gui.font_data.data(), static_cast<int>(font_file_size), 16, &font_config, io.Fonts->GetGlyphRangesJapanese());
 }
 
-static void init_background(State &gui, const std::string &image_path) {
+void init_background(HostState &host, const std::string &image_path) {
```

Comment:
Does `init_background` use `HostState` or just `gui::State`? Not too sure why this change was made.

Reply:
For get works on settings dialog ^^ and use like this same other works perfectly